### DISCUSSION TITLE
Question: Running Kwok as a Kubernetes Job

### DISCUSSION THREAD

**Discussion by helloJamest** [kubernetes-sigs/kwok#538] [Q&A] [answered]
Our goal is to develop the Kwok simulator as a Kubernetes Job so that we can deploy it into Kubernetes and execute the simulator whenever we need it. This will enable us to easily manage and scale the simulator based on our requirements.

**Comment by wzshiming** [comment, marked as answer]
Did you mean to install KWOK in a cluster? 

https://kwok.sigs.k8s.io/docs/user/kwok-in-cluster/
